packages/kernel-browser-runtime/src/kernel-worker/kernel-worker.ts

```diff
@@ -42,12 +42,18 @@ async function main(): Promise<void> {
       PlatformServicesClient.make(globalThis as PostMessageTarget),
       makeSQLKernelDatabase({ dbFilename: DB_FILENAME }),
     ]);
+
   const firstTime = !kernelDatabase.kernelKVStore.get('initialized');
+  const resetStorage =
+    globalThis.location.search.includes('reset-storage=true');
```

Comment:
This feels like a sketchy way to check the query string.  I know this is just debug hackery, but...

Reply:
Dang, that was quick!

Reply:
The robot pre-empted you (good robot)